apps/examples/testcase/le_tc/kernel/kernel_tc_main.c

```diff
@@ -100,7 +100,7 @@ int kernel_tc_main(int argc, char *argv[])
 
 #ifdef CONFIG_TC_KERNEL_LIBC_PTHREAD
 #if (!defined CONFIG_PTHREAD_MUTEX_TYPES)
-#error CONFIG_PTHREAD_MUTEX_TYPES is needed for testing LIBC_PTHREAD TC
+	/* #error CONFIG_PTHREAD_MUTEX_TYPES is needed for testing LIBC_PTHREAD TC */
```

Comment:
Ok.
and for your below query,
```
le_tc/kernel/tc_pthread.c:1097:55: error: 'pthread_mutexattr_t' has no member named 'robust'
  TC_ASSERT_EQ("pthread_mutex_init", mutex.flags, (attr.robust == PTHREAD_MUTEX_ROBUST ? _PTHREAD_MFLAGS_ROBUST : 0));
```
Below solution should resolve it and please review the same.

In `os/include/sched.h +299`
```
#if defined(CONFIG_PTHREAD_MUTEX_BOTH) || defined(PTHREAD_MUTEX_ROBUST)
        uint8_t robust  : 1; /* PTHREAD_MUTEX_STALLED or PTHREAD_MUTEX_ROBUST */
#endif
```